packages/perseus/src/widgets/explanation.tsx

```diff
@@ -91,9 +91,17 @@ class Explanation extends React.Component<Props, State> {
         // Special styling is needed to fit the button in a block of text without throwing off the line spacing.
         // While the button is not normally included in a block of text, it needs to be able to accommodate such a case.
         const buttonStyleOverrides = {
-            height: "22px",
+            height: "auto",
             marginLeft: "-2px",
-            padding: "0 2px",
+            marginRight: "2px",
+            paddingLeft: "2px",
+        };
+
+        const labelStyle = {
+            fontSize: "18px",
+            lineHeight: "20px",
+            marginRight: "-6px",
```

Comment:
What does this negative margin do? It feels a bit strange that we are adding `marginRight: 2px` to the button but then removing margin from the label. Might be worth a comment explaining why we do this.

Reply:
Comments added.